src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -194,6 +225,40 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetEnableCallStackCollectionOnThreadStateChanges(bool check) {
+  ui_->threadStateChangeCallstackCollectionCheckBox->setChecked(check);
+}
+
+bool CaptureOptionsDialog::GetEnableCallStackCollectionOnThreadStateChanges() const {
+  return ui_->threadStateChangeCallstackCollectionCheckBox->isChecked();
+}
+
+void CaptureOptionsDialog::SetThreadStateChangeCallstackMethod(
+    CaptureOptions::UnwindingMethod thread_state_change_callstack_method) {
+  switch (thread_state_change_callstack_method) {
+    case CaptureOptions::kFramePointers:
+      ui_->threadStateChangeCallstackFramepointersMethodRadioButton->setChecked(true);
+      ui_->threadStateChangeCallstackDWARFMethodRadioButton->setChecked(false);
+      break;
+    case CaptureOptions::kDwarf:
+    default:
+      ui_->threadStateChangeCallstackDWARFMethodRadioButton->setChecked(true);
```

Comment:
Nit: Swap these lines to be consistent with the previous case.

```
 ui_->threadStateChangeCallstackFramepointersMethodRadioButton->setChecked(false);
 ui_->threadStateChangeCallstackDWARFMethodRadioButton->setChecked(true);
```

Reply:
Done.